### PR TITLE
chore: node readiness

### PR DESCRIPTION
**Pull request by romanzac**
Update node readiness check. Add /health endpoint usage.

### CODE REVIEW

File: src/node/waku_node.py [hunk 1]
```diff
@@ -188,17 +188,34 @@ def unpause(self):
 
     def ensure_ready(self, timeout_duration=10):
         @retry(stop=stop_after_delay(timeout_duration), wait=wait_fixed(0.1), reraise=True)
+        def check_healthy(node=self):
+            self.health_response = node.health()
+            if self.health_response == b"Node is healthy":
+                logger.info("Node is healthy !!")
+                return
+            else:
```
Comment (AlejandroCabeza) [inline src/node/waku_node.py:196]: Styling: Given the `if` arm contains a `return`, I wouldn't use an `else`. But this is a styling preference, up to you :)

File: src/node/waku_node.py [hunk 2]
```diff
@@ -188,17 +188,34 @@ def unpause(self):
...
+                try:
+                    self.health_response = json.loads(self.health_response)
+                except Exception as ex:
+                    raise AttributeError(f"Unknown health response format {ex}")
+
+            if self.health_response["nodeHealth"] and self.health_response["nodeHealth"] != "Ready":
```
Comment (AlejandroCabeza) [inline src/node/waku_node.py:202]: The few checks in here that follow the format `self.health_response["nodeHealth"] and self.health_response["nodeHealth"] != "Ready"` can be replaced by the simpler:
```py
self.health_response.get("nodeHealth") != "Ready"
```

File: src/node/waku_node.py (hunk 1 above)
Comment (romanzac) [reply src/node/waku_node.py]: You are right, I just have only one "else". Fixed at https://github.com/waku-org/waku-interop-tests/pull/35/commits/c61aef2da37d485ea53c93e453e3c1c5aa139289

File: src/node/waku_node.py (hunk 2 above)
Comment (romanzac) [reply src/node/waku_node.py]: I have wrongly assumed, checking step by step won't lead to key error, if the key is missing.  Fixed at https://github.com/waku-org/waku-interop-tests/pull/35/commits/c61aef2da37d485ea53c93e453e3c1c5aa139289

**Review by AlejandroCabeza — APPROVED** [review]

**Pull request merged**